runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -354,22 +364,73 @@ class TR_VectorAPIExpansion : public TR::Optimization
 
 
   /** \brief
-   *     Checks if method's argument is one the \c vapiObjType types
+   *     Returns index of a child node that contains element type
    *
    *  \param methodSymbol
    *     Method symbol
    *
-   *  \param i
-   *     argument's number
+   *  \return
+   *     index of a child node that contains element type
```

Comment:
Please change `index` -> `Index`, following the descriptions of other functions.